imap_processing/codice/l0/decompress_codice.py

```diff
@@ -0,0 +1,111 @@
+"""Decompress CoDICE science data.
+
+For CoDICE, there are 3 forms of compression:
+    1. Table-based lossy compression A (24 -> 8-bit)
+    2. Table-based lossy compression B (24 -> 8 bit)
```

Comment:
to be consistent with below functions, should we change this or point out that it could change.
```suggestion
    1. Table-based lossy compression A (32 -> 8-bit)
    2. Table-based lossy compression B (32 -> 8 bit)
```